### PR TITLE
add: uiHooks type definition #476

### PR DESCRIPTION
**Pull request by ckcks12**
#476 

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by climba03003 — CHANGES_REQUESTED** [review]
Can you add an unit test for the new types?

File: index.d.ts [hunk 1]
```diff
@@ -60,6 +60,13 @@ export interface FastifySwaggerOptions {
    */
   staticCSP?: boolean | string | Record<string, string | string[]>
   transformStaticCSP?: (header: string) => string
+  /**
+   * route hooks
+   */
+  uiHooks: {
```
Comment (climba03003) [inline index.d.ts:66]: It should be optional.
```suggestion
  uiHooks?: {
```

**Review by climba03003 — CHANGES_REQUESTED** [review]

**Comment by ckcks12** [comment]
You mean here? 

https://github.com/fastify/fastify-swagger/blob/master/test/types/types.test.ts

**Comment by climba03003** [comment]
> 
> 
> You mean here?
> 
> https://github.com/fastify/fastify-swagger/blob/master/test/types/types.test.ts

Yes

**Comment by ckcks12** [comment]
done :)

**Review by climba03003 — APPROVED** [review]
LGTM.

**Pull request merged**